spec/models/otp_request_spec.rb

```diff
@@ -7,7 +7,7 @@
 
     context 'When mobile number and otp are of exactly 10 and 6 digits respectively' do
       it 'should be valid' do
-        otp = OtpRequest.new(mobile_number: valid_mobile_number, otp: valid_otp, expire_at: 10.minutes.from_now)
+        otp = build(:otp_request, mobile_number: valid_mobile_number, otp: valid_otp, expire_at: 10.minutes.from_now)
```

Comment:
Optional: We could use the [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers) gem to write easier tests. It would help us check many things about our model in a simpler way. 

For example, we could write: `it { should validate_presence_of(:mobile_number) }`

Reply:
My initial thought process was to be very much explicit on what is being tested. But I think it makes sense to have shoulda_matchers as it will be too much to have custom specs added for every new association/validations added in the future. 

Added the gem and updated the specs